client/src/components/exercise/lab9/hooks/useRepairNav.js

```diff
@@ -1,9 +1,81 @@
+import { NavBarData } from "../../../../constants/lab9/NavBarData";
```

Comment:
Delete this file and use the `useDataService` hook in the the Nav Bar page component